src/winetricks

```diff
@@ -12776,8 +12776,8 @@ w_metadata steam apps \
 
 load_steam()
 {
-    # 2016/10/28: 029f918a29b2b311711788e8a477c8de529c11d7dba3caf99cbbde5a983efdad
```

Comment:
Thanks for the PR. The old date/sha should remain, see e.g., flash for an example.